src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests.csproj

```diff
@@ -0,0 +1,46 @@
+<Project>
+  <PropertyGroup>
+    <TargetFrameworks>net472;$(SdkTargetFramework)</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(DotNetBuildFromSource)' == 'true' ">net5.0</TargetFrameworks>
+    <AssemblyName>Microsoft.NET.Sdk.Publish.Tasks.Tests</AssemblyName>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <EndToEndTestsDisabled>true</EndToEndTestsDisabled>
```

Comment:
This is what is causing the E2E tests to be disabled.

Reply:
Let me try running it locally to see if there are any issues, else we can enable it. I had disabled it for the net5.0 migration.

Reply:
Yep, just matching your change. May want to get this in first, then reenable.

Reply:
yup. Agreed. hence the PR approval.